### PR TITLE
Add new BBs

### PR DESCRIPTION
**Pull request by LS0205**
Added the BBs and Usecases proposed at the Federate Networking Event, from the 23.9.2024

### CODE REVIEW

File: BB-CEST/BB_Car_Simulator.md [hunk 1]
```diff
@@ -7,11 +7,11 @@ BB-CEST
...
 <!-- In which Functional Cluster the BB be located; if none of the existing fit new required -->
-None
+Not Specified
 
 ## Layer
 <!-- AppLayer, MWLayer, OSLayer, HWLayer -->
-Cross-Layer
+Not Specified
 
 ## Known Implementation
```
Comment (AnotherDaniel) [inline BB-CEST/BB_Car_Simulator.md:17]: Carla (https://carla.org/)

File: BB-CSC/BB_Cloud_container_and_OS.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+## Functional Clusters
+<!-- In which Functional Cluster the BB be located; if none of the existing fit new required -->
+Not specified
+
+## Layer
+<!-- AppLayer, MWLayer, OSLayer, HWLayer -->
+Not specified
+
+## Known Implementation
+
```
Comment (AnotherDaniel) [inline BB-CSC/BB_Cloud_container_and_OS.md:17]: - Blackberry QNX
- OxidOS
- Eclipse ThreadX
- RedHat In-Vehicle OS (announced)

File: BB-CSC/BB_Data_spaces.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+## Functional Clusters
+<!-- In which Functional Cluster the BB be located; if none of the existing fit new required -->
+Not specified
+
+## Layer
+<!-- AppLayer, MWLayer, OSLayer, HWLayer -->
+Not specified
+
+## Known Implementation
+
```
Comment (AnotherDaniel) [inline BB-CSC/BB_Data_spaces.md:17]: - Catena-X?

File: BB-CSC/BB_End_to_end_security_for_C2X.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+## Functional Clusters
+<!-- In which Functional Cluster the BB be located; if none of the existing fit new required -->
+Not specified
+
+## Layer
+<!-- AppLayer, MWLayer, OSLayer, HWLayer -->
+Not specified
+
+## Known Implementation
+
```
Comment (AnotherDaniel) [inline BB-CSC/BB_End_to_end_security_for_C2X.md:17]: - all current state-of-the-art IT security protocols and practices

File: BB-CSC/BB_Extended_accident_data_recorder.md [hunk 5]
```diff
@@ -0,0 +1,98 @@
+
+# Extended acciendet data recorder
```
Comment (AnotherDaniel) [inline BB-CSC/BB_Extended_accident_data_recorder.md:2]: "accident"

File: BB-EST/Testing/BB_OpenDUT.md
Comment (AnotherDaniel) [inline BB-EST/Testing/BB_OpenDUT.md:1]: This is not a building block, but a specific implementation that incorporates many (abstract) BBs. Also see #36

File: BB-CSC/BB_Orchestration_support_for_energy_and_mobility_providers.md
Comment (AnotherDaniel) [inline BB-CSC/BB_Orchestration_support_for_energy_and_mobility_providers.md:1]: I do not think that "energy and mobility providers" would need (or get) a different orchestration infrastructure from anyone else...

File: S-BB/MWLayer/BB_Standardized_HD_maps_formats.md [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+
+## Functional Clusters
+<!-- In which Functional Cluster the BB be located; if none of the existing fit new required -->
+
+## Layer
+<!-- AppLayer, MWLayer, OSLayer, HWLayer -->
+MWLayer
+
+## Known Implementation
+
```
Comment (AnotherDaniel) [inline S-BB/MWLayer/BB_Standardized_HD_maps_formats.md:16]: - HERE (https://www.here.com)

File: S-BB/MWLayer/BB_Standardized_cloud_services_to_support_international_roaming_of_vehicles.md
Comment (AnotherDaniel) [inline S-BB/MWLayer/BB_Standardized_cloud_services_to_support_international_roaming_of_vehicles.md:1]: What exactly is the idea here - what is the delta to existing globally scaled data centers, location-aware content delivery and mobile network roaming capabilities?

**Review by AnotherDaniel — APPROVED** [review]
Next time, it would be better to have different PRs for new  BBs vs adding the identical maturity section to existing ones.

File: BB-CEST/BB_Car_Simulator.md (hunk 1 above)
Comment (LS0205) [reply BB-CEST/BB_Car_Simulator.md:17]: added

File: BB-CSC/BB_Cloud_container_and_OS.md (hunk 2 above)
Comment (LS0205) [reply BB-CSC/BB_Cloud_container_and_OS.md:17]: added

File: BB-CSC/BB_Data_spaces.md (hunk 3 above)
Comment (LS0205) [reply BB-CSC/BB_Data_spaces.md:17]: added

**Review by madri2 — DISMISSED** [review]
fine for me

**Review by prillerp — APPROVED** [review]
all updates were already reviewed by Daniel and Mario; new PR due to re-basing.

**Pull request merged**